.github/workflows/pr-open-workflow.yml

```diff
@@ -0,0 +1,28 @@
+# SPDX-FileCopyrightText: 2022 The Aalto Grades Developers
+#
+# SPDX-License-Identifier: MIT
+
+name: PR Open Workflow
+run-name: ${{ github.actor }} pull request
+on:
+  pull_request:
+    types: [opened, ready_for_review]
+    branches:
+      - main
+    
+jobs:
+  notify_telegram:
+    name: Notify Telegram On PR Open
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send PR message to telegram
+        uses: appleboy/telegram-action@master

```

Comment:
At [GitHub Marketplace](https://github.com/marketplace/actions/telegram-message-notify), the latest version is v0.1.1. Could we bind to this version to avoid (admittedly unlikely) breakage?